package.json

```diff
@@ -18,8 +18,8 @@
     "cypress:backend": "cd backend && yarn run dev",
     "cypress:webapp": "cd webapp && yarn run dev",
     "cypress:setup": "run-p cypress:backend cypress:webapp",
-    "cypress:run": "cross-env cypress run",
-    "cypress:open": "cross-env cypress open",
+    "cypress:run": "cross-env cypress run --browser firefox",
+    "cypress:open": "cross-env cypress open --browser firefox",
```

Comment:
That has the drawback that we cannot record our cypress tests anymore :cry: can we close?

Reply:
No, actually it doesn't, if you look at the build, or click on cypress, you can see that all the videos were recorded

Reply:
I wouldn't close, is there any justifiable reason to use a different browser than what we recommend our users to use for e2e tests?